src/infection-model/index.js

```diff
@@ -0,0 +1,122 @@
+// this function comes from Justine, adapted from python model
+function simulateOneDay({
+  x,
+  N,
+  R0,
+  dIncubation = 2,
+  dInfectious = 4.1,
+  dRecoveryMild = 9.9,
+  dHospitalLag = 2.9,
+  dHospitalRecovery = 22,
+  dHospitalFatality = 8.3,
+  pSevereCase = 0.26,
+  pFatalityRate = 0.026,
+}) {
+  // Compute the updated SEIR values after 1 day
+  // x: [1D array of integers] counts for the 9 SEIR (susceptible/exposed/infectious/recovered) values
+  // N: [integer] population total per age bucket
+  // R0: [float] rate of spread factor
+  // D_incubation: [float] days from virus exposure to infectious/contagious state
+  // D_infectious: [float] days in infectious period
+  // D_recovery_mild: [float] days from end of infectious period to end of virus
+  // D_hospital_lag: [float] days from end of infectious period to hospital admission
+  // D_hospital_recovery: [integer] days from hospital admission to hospital release (non-fatality scenario)
+  // D_hospital_fatality: [float] days from hospital admission to deceased (fatality scenario)
+  // P_severe_case: [float] probability case will be severe enough for the hospital
+  // P_fatality_rate: [float] probability of mortality per age bucket
+
+  const alpha = 1 / dIncubation;
+  const beta = R0 / dInfectious;
+  const gamma = 1 / dInfectious;
+
+  const pMildCase = 1 - pSevereCase;
+
+  const susceptible = x[0];
+  const exposed = x[1];
+  const infectious = x[2];
+  const mild = x[3];
+  const severe = x[4];
+  const hospitalized = x[5];
+  const mildRecovered = x[6];
+  const severeRecovered = x[7];
+  const fatalities = x[8];
+
+  // Compute the deltas from the bottom of the tree up, additions to the leaf nodes are subtracted from their parent node as people flow from the root to the leaves of the tree
+  const mildRecoveredData = mild / dRecoveryMild;
+  const fatalitiesDelta =
+    ((pFatalityRate / pSevereCase) * hospitalized) / dHospitalFatality;
+  const severeRecoveredDelta =
+    ((1 - pFatalityRate / pSevereCase) * hospitalized) / dHospitalRecovery;
```

Comment:
Should it actually be `(((1 - pFatalityRate) / pSevereCase) * hospitalized) / dHospitalRecovery`? That is, instead of the math for the first component being `1 - (pFatalityRate / pSevereCase)` as it currently is, should it be `(1 - pFatalityRate) / pSevereCase`? 

Looking at it more closely, I think it actually _should_ be how you have it now, that the first component is supposed to be 1 - the first component from the previous calculation. But extra parentheses for grouping may broadcast intention more clearly? Take or leave that comment as you see fit.

Reply:
when I add the extra parens Prettier auto-deletes them so ........... this is what it's gonna be